pkg/authorization/local_authorizer.go

```diff
@@ -80,5 +93,14 @@ func (a *LocalAuthorizer) Authorize(ctx context.Context, attr authorizer.Attribu
 		&rbac.ClusterRoleBindingLister{Lister: filteredInformer.ClusterRoleBindings().Lister()},
 	)
 
-	return scopedAuth.Authorize(ctx, attr)
+	dec, reason, err := scopedAuth.Authorize(ctx, attr)
+
+	decString := decisionString(dec)
```

Comment:
global comment: if you're not reusing this, you can omit creating a variable

Reply:
hrm, yes, this is a left-over from a previous code state, this var doesn't make sense any more.

Reply:
fixed